core/src/main/resources/xsl/checker2dot.xsl

```diff
@@ -218,7 +218,7 @@
                  <xsl:if test="not($nfaMode) and exists(@priority)">
                      <xsl:value-of select="concat('\n[',@priority,']')"/>
                  </xsl:if>
-                 <xsl:value-of select="'&quot;'"/>
+                 <xsl:value-of select="concat('\n',@id,'&quot;')"/>
```

Comment:
Where would be a better place to be able to have the ID show up with the `-e` and `-p ` options?